service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/enums/DocumentSectionCode.java

```diff
@@ -0,0 +1,15 @@
+package uk.nhs.adaptors.oneoneone.cda.report.enums;
```

Comment:
we may not need this

Reply:
populate codeable concept as its done in other classes, keep it consistent

Reply:
Done